File: javascript/get_post.js
var data = null;

var xhr = new XMLHttpRequest();
xhr.withCredentials = true;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === this.DONE) {
    console.log(this.responseText);
  }
});

xhr.open("GET", "https://graph.microsoft.com/v1.0/groups/%7Bid%7D/threads/%7Bid%7D/posts/%7Bid%7D");

xhr.send(data);

File: ruby/get_post.rb
require 'uri'
require 'net/http'

url = URI("https://graph.microsoft.com/v1.0/groups/%7Bid%7D/threads/%7Bid%7D/posts/%7Bid%7D")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true
http.verify_mode = OpenSSL::SSL::VERIFY_NONE

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body
